pkg/scalers/selenium_grid_scaler.go

```diff
@@ -29,72 +28,86 @@ type seleniumGridScaler struct {
 type seleniumGridScalerMetadata struct {
 	triggerIndex int
 
-	URL                   string `keda:"name=url,                     order=triggerMetadata;authParams"`
-	BrowserName           string `keda:"name=browserName,             order=triggerMetadata"`
-	SessionBrowserName    string `keda:"name=sessionBrowserName,      order=triggerMetadata, optional"`
-	ActivationThreshold   int64  `keda:"name=activationThreshold,     order=triggerMetadata, optional"`
-	BrowserVersion        string `keda:"name=browserVersion,          order=triggerMetadata, optional, default=latest"`
-	UnsafeSsl             bool   `keda:"name=unsafeSsl,               order=triggerMetadata, optional, default=false"`
-	PlatformName          string `keda:"name=platformName,            order=triggerMetadata, optional, default=linux"`
-	SessionsPerNode       int64  `keda:"name=sessionsPerNode,         order=triggerMetadata, optional, default=1"`
-	SetSessionsFromHub    bool   `keda:"name=setSessionsFromHub,      order=triggerMetadata, optional, default=false"`
```

Comment:
These options have been added in the last commit and this removes that feature :)

Reply:
In my change it will cover via - Added one more parameter nodeMaxSessions (default is 1), to configure the scaler to be aligned with option --max-sessions (env var SE_NODE_MAX_SESSIONS) in the Node

Reply:
I also added a few unit tests that are similar to that feat, still checking if I can optimize this since I try to archive reduce parameters needed but do the same expectation

Reply:
@JorTurFer, I added UT from that PR. All UTs passed now. I think it's ready to go.